julia/test/model_tests.jl

```diff
@@ -651,6 +651,9 @@ end
     using LinearAlgebra
     @test isapprox(-Matrix(1.0I, D, D), hess)
 
+    v = rand(D)
+    lp, Hvp = BridgeStan.log_density_hessian_vector_product(model, y, v)
+    @test isapprox(-v, Hvp)
```

Comment:
Do we have `BRIDGESTAN_AD_HESSIAN` defined for one interface language and not defined for another language, such that both C paths are tested?

Reply:
Yes, the Python interface tests run some tests with both BRIDGESTAN_AD_HESSIAN and without